webapp/niapp.yaml

```diff
@@ -122,6 +125,29 @@ paths:
           $ref: '#/responses/CreateWebAppResponse'
         401:
           $ref: '#/responses/Unauthorized'
+  '/webapps/query-webapps':
```

Comment:
Shouldn't that be just /webapps/query

/webapps/query-webapps seems redundant...

Reply:
I've updated the documentation here. Once this is submitted, I'll also update the code in the webapp service.